client/context.go

```diff
@@ -329,27 +329,32 @@ func GetFromFields(kr keyring.Keyring, from string, genOnly bool) (sdk.AccAddres
 		return addr, "", 0, nil
 	}
 
-	var info keyring.Info
+	k := new(keyring.Record) //nolint
```

Comment:
I put `nolint` to resolve liner issue `k is not used in code`

Reply:
OK, this is weird, because `k` is obviously used in  L354